package.json

```diff
@@ -25,15 +25,22 @@
         "test:migration": "cd backend && npm run test:migration",
         "test": "cd shared && npm run test && cd ../frontend && npm run test && cd ../backend && npm run test",
         "migration:run:prod": "cd backend && npm run migration:run:prod",
-        "merge-coverage": "mkdir coverage && cp backend/coverage/coverage-final.json coverage/backend.json && cp frontend/coverage/coverage-final.json coverage/frontend.json && cp shared/coverage/coverage-final.json coverage/shared.json && nyc merge coverage coverage/merged/coverage.json && nyc report -t coverage/merged --report-dir coverage/reports --reporter=html --reporter=cobertura --reporter=text --reporter=text-summary"
+        "merge-coverage": "mkdir coverage && cp backend/coverage/coverage-final.json coverage/backend.json && cp frontend/coverage/coverage-final.json coverage/frontend.json && cp shared/coverage/coverage-final.json coverage/shared.json && nyc merge coverage coverage/merged/coverage.json && nyc report -t coverage/merged --report-dir coverage/reports --reporter=html --reporter=cobertura --reporter=text --reporter=text-summary",
+        "licensing:third-party-licenses": "npm-run-all -s -n -c licensing:third-party-licenses:json licensing:third-party-licenses:html",
+        "licensing:third-party-licenses:json": "license-reporter --output frontend/src/assets/source-code/THIRD-PARTY-LICENSE-ACKNOWLEDGEMENTS.json",
+        "licensing:third-party-licenses:html": "node third-party-license-json-to-html.cjs",
+        "licensing:source-code-zip-file": "git archive --output=./frontend/src/assets/source-code/digital-fuesim-manv.zip --format=zip HEAD",
+        "licensing:all": "npm-run-all -s -n -c licensing:third-party-licenses licensing:source-code-zip-file"
```

Comment:
`npm-run-all` does not seem to be part of a normal Node.js+npm installation. Could you please add it to the devDependencies?

By the way: The `package-lock.json` of this branch does not match the installed packages. On the first glance, it seemed like this way only the `npm-run-all` package wich was part of your lockfile, but not the `package.json` (maybe because you've used `npx` once?), but please make sure the lockfile is up-to-date with the packages from the `package.json` after adding the missing package.

Reply:
@lukasrad02 I have installed it, is something else missing?

Reply:
Looks good now. There are no missing packages and the lockfiles are up-to-date